models/go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/aws/aws-sdk-go/models
+
+go 1.13
+
+require (
+	github.com/aws/aws-sdk-go v1.29.0
+	github.com/aws/aws-sdk-go/private/model v0.0.0-00010101000000-000000000000
```

Comment:
Shouldn't this be reversed? Should `private/model` depend on this `model` package instead?